assets/styles/tools/_maths.scss

```diff
@@ -0,0 +1,134 @@
+// ==========================================================================
+// Tools / Maths
+// ==========================================================================
+
+// Removes the unit from the given number.
+//
+// @param  {number} $number The number to strip.
+// @return {number}
+
+@function strip-units($number) {
+    @return $number / ($number * 0 + 1);
+}
+
+// Square root function
+//
+// @param  {string} $number     The number
+// @return {string}             The sqrt number
+
+@function sqrt($number) {
+    $x: 1;
+    $value: $x;
+
+    @for $i from 1 through 10 {
+        $value: $x - ($x * $x - abs($number)) / (2 * $x);
+        $x: $value;
+    }
+
+    @return $value;
+}
+
+// Power function
+//
+// @param  {number} $number Number to apply power
+// @param  {number} $exp    The exponant for the power
+// @return {number}         The powered number
+
+@function pow($number, $exp) {
+    $value: 1;
+
+    @if $exp > 0 {
+        @for $i from 1 through $exp {
+            $value: $value * $number;
+        }
+    } @else if $exp < 0 {
+        @for $i from 1 through -$exp {
+            $value: $value / $number;
+        }
+    }
+
+    @return $value;
+}
+
+// Factorial function
+//
+// @param  {number} $number The number to factorize
+// @return {number}         The factorised number
+
+@function fact($number) {
+    $value: 1;
+
+    @if $number > 0 {
+        @for $i from 1 through $number {
+            $value: $value * $i;
+        }
+    }
+
+    @return $value;
+}
+
+// Pi reference
+//
+// @return {number} Retrun PI with 11 decimals
+
+@function pi() {
+    @return 3.14159265359;
+}
+
+// Convert deg to rad
+//
+// @param  {string} $angle The angle to convert
+// @return {number}        The unitless angle converted to rad
```

Comment:
```suggestion
// Converts the number in degrees to the radian equivalent .
//
// @param  {number} $angle The angular value to calculate.
// @return {number}        If $angle has the `deg` unit,
//     the radian equivalent is returned.
//     Otherwise, the unitless value of $angle is returned.
```